client/components/widget/query/query-result-data-service.js

```diff
@@ -44,6 +44,8 @@ const WorkQueueService = explorer.components.util.WorkQueueService;
 const WidgetConfig = explorer.models.WidgetConfig;
 
 
+const ERR_UNEXPECTED = 'An unexpected error occurred.';
```

Comment:
Added prefix text to all error messages (An error occurred when fetching data from FOO), and rewrote ERR_UNEXPECTED to focus on the HTTP response's lack of detail.